egs/librimix/ConvTasNet/eval.py

```diff
@@ -1,25 +1,22 @@
+import argparse
+import json
 import os
 import random
+from pprint import pprint
+
+import pandas as pd
 import soundfile as sf
 import torch
 import yaml
-import json
-import argparse
-import pandas as pd
 from tqdm import tqdm
-from pprint import pprint
-import numpy as np
```

Comment:
I don't understand that part of the changes? Only numpy is not needed? 
Why did the ordering changes and their is a new line?

Reply:
To be honest I just applied the import optimizer of pycharm i guess it groups imports and order them. 
it removes `from model import make_model_and_optimizer` `import numpy as np`  `from asteroid import torch_utils`. 
I can keep the original ordering if you wish.

Reply:
Yes, please, git history is clearer in this way. If we want to optimize imports or refactor code or anything, this should be a separate PR, that does only this. And alphabetizing imports doesn't seem important to me.